### PR TITLE
AlTaieu MobIDs and isSpawned() checks

### PR DESCRIPTION
**Pull request by wrenffxi**

### CODE REVIEW

File: scripts/zones/AlTaieu/mobs/Aw_euvhi.lua [hunk 1]
```diff
@@ -5,8 +5,9 @@
 package.loaded["scripts/zones/AlTaieu/TextIDs"] = nil;
 -----------------------------------
 
-require("scripts/zones/AlTaieu/TextIDs");
 require("scripts/globals/keyitems");
+require("scripts/zones/AlTaieu/MobIDs");
+require("scripts/zones/AlTaieu/TextIDs");
```
Comment (TeoTwawki) [inline scripts/zones/AlTaieu/mobs/Aw_euvhi.lua]: *sniffle*

reversing my upside down pyramids of headers again

😢

File: scripts/zones/AlTaieu/mobs/Aw_euvhi.lua [hunk 2]
```diff
@@ -28,21 +29,24 @@ end;
...
-    elseif (mobID == 16912817 or mobID == 16912821 or mobID == 16912819) then
-        if (player:hasKeyItem(RED_CARD) == false) then
+
+    elseif (mobID == EUVHIS_RED+0 or mobID == EUVHIS_RED+2 or mobID == EUVHIS_RED+4) then
+        if (not player:hasKeyItem(RED_CARD)) then
             player:addKeyItem(RED_CARD);
             player:messageSpecial(KEYITEM_OBTAINED,RED_CARD);
         end
+
+    elseif (mobID == EUVHIS_BLACK+0 or mobID == EUVHIS_BLACK+2 or mobID == EUVHIS_BLACK+4) then
```
Comment (TeoTwawki) [inline scripts/zones/AlTaieu/mobs/Aw_euvhi.lua]: plus zero?

Comment (TeoTwawki) [reply scripts/zones/AlTaieu/mobs/Aw_euvhi.lua]: *nudge*

> plus zero?

I can has reason?

Comment (wrenffxi) [reply scripts/zones/AlTaieu/mobs/Aw_euvhi.lua]: EUVHIS_BLACK is base value.  +0, +2, and +4 are offsets from base.  Figured the consistency would make it obvious we're offsetting from a starting value to reach a final mobId.

Comment (TeoTwawki) [reply scripts/zones/AlTaieu/mobs/Aw_euvhi.lua]: k

File: scripts/zones/AlTaieu/mobs/Aw_euvhi.lua (hunk 1 above)
Comment (wrenffxi) [reply scripts/zones/AlTaieu/mobs/Aw_euvhi.lua]: Early birthday present.

Comment (TeoTwawki) [reply scripts/zones/AlTaieu/mobs/Aw_euvhi.lua]: didnt hafta but I lurva ya for it<3

**Pull request merged**